(node:592768) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-nosession-43336',
  containerIp: '10.0.3.251',
  mitmPort: 43336,
  autoAccessFixed: 1
}
2024-11-12 00:37:32.528 - [Info] MITM Version: 2
2024-11-12 00:37:32.530 - [Info] Auto Access Enabled: true
2024-11-12 00:37:32.531 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-12 00:37:32.564 - [Info] SSH man-in-the-middle server for 10.0.3.251 listening on 10.0.3.1:43336
2024-11-12 00:38:05.063 - [Debug] [Connection] Attacker connected: 203.129.225.196 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-12 00:38:06.696 - [Debug] [Auth] Attacker 203.129.225.196 trying to authenticate with 'password'
2024-11-12 00:38:06.697 - [Debug] [Auto Access] Attacker: 203.129.225.196, Threshold: 1, Attempts: 1
2024-11-12 00:38:06.697 - [Debug] [Auto Access] Compromising the honeypot
2024-11-12 00:38:06.697 - [Debug] [Auto Access] Adding the following credentials: 'deamon:deamon'
2024-11-12 00:38:06.988 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-12 00:38:06.991 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-12 00:38:06.991 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.251
2024-11-12 00:38:07.085 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-12 00:38:08.221 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-12 00:38:09.884 - [Debug] [Connection] Attacker connected: 203.129.225.196 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-12 00:38:11.510 - [Debug] [Auth] Attacker 203.129.225.196 trying to authenticate with 'password'
2024-11-12 00:38:11.510 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-12 00:38:11.511 - [Debug] [Auth] Attacker: 203.129.225.196 has so far made 1 attempts. Remaining: 5 attempts
2024-11-12 00:38:12.078 - [Debug] [Connection] Attacker closed the connection
2024-11-12 00:38:12.352 - [Debug] [Connection] Attacker connected: 203.129.225.196 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
